scripts/test_line_bot.py

```
@@ -0,0 +1,233 @@
+        
+    except Exception as e:
+        print(f"❌ メッセージ処理テストエラー: {e}")
+    
+    finally:
+        if 'mongo_client' in locals():
+            await mongo_client.disconnect()
+
+
+def test_utility_functions():
+    """ユーティリティ関数のテスト"""
+    print(f"\n🔧 ユーティリティ関数テスト")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)